README.md

```diff
@@ -265,6 +265,58 @@ const roleId = '000000000000-0000-0000-0000-00000000'
 await accountClient.deleteRealmRole(realmName, roleId)
 ```
 
```

Comment:
Not sure if we should add additional documentation in this section explicitly calling out that 'Realm Application Role' CRUD uses the role name and not an id like 'Realm Role'? or just let the examples speak for themselves?

Reply:
ah, lovely 😆

i'm happy to let the examples & function signatures speak for themselves.